portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase

```diff
@@ -1107,6 +1107,577 @@ definition {
+                dataSetViewDescription = "FDSViewDescription",
+                key_dataSetViewNameList = "View Fields Test");
+
+            DataSetAdmin.goToSpecificViewPage(
+                dataSetName = "Dataset Test",
+                key_viewName = "View Fields Test");
+        }
+
+        task ("When goes to Fields Tab.") {
+            DataSetAdmin.goToTab(tabName = "Filters");
+
+            LexiconEntry.gotoAdd();
+
+            Click(
+                key_fieldName = "Selection",
+                locator1 = "FormViewBuilder#OBJECT_FIELD_BUTTON_DROPDOWN");
+
+            Type(
+                key_text = "Name",
+                locator1 = "TextInput#ANY",
+                value1 = "Bike");
+
+            Click(
+                key_class = "btn-secondary",
+                key_index = 2,
+                locator1 = "Button#ANY_WITH_CLASS_AND_INDEX");
+
+            SelectFrameTop();
+
+            Click.javaScriptClick(
+                key_sortOption = "Relationship_c_fdsViewERC",
+                locator1 = "ObjectCustomViews#SORT_OPTION");
+
+            Select(
+                locator1 = "DataSet#SOURCE_OPTIONS_SELECT",
+                value1 = "Transport");
+        }
+
+        task ("And Preselected Values bike") {
+            Click(
+                key_fieldName = "Select a default value for your filter.",
+                locator1 = "ObjectCustomViews#FROM_DATE_INPUT");
+
+            Click(
+                key_sortOption = "bike",
+                locator1 = "ObjectCustomViews#SORT_OPTION");
+        }
+
+        task ("And Select the option Multiple,Include radio button") {
+            Click.clickAtNotVisible(
+                key_radioOption = "Multiple",
+                locator1 = "CommerceEntry#RADIO_BUTTON");
+
+            Click.clickAtNotVisible(
+                key_radioOption = "Include",
+                locator1 = "CommerceEntry#RADIO_BUTTON");
+        }
+
+        task ("And the user save the change clicking on save button") {
+            Button.clickSave();
+        }
+
+        task ("When the user adds a widget page and add a fragment Data Set") {
+            Navigator.openURL();
+
+            ContentPages.addPage(pageName = "Test Page");
+
+            PageEditor.addFragment(
+                collectionName = "Content Display",
+                fragmentName = "Data Set");
+        }
+
+        task ("And viewing the configuration menu of the portlet And selects View Fields Test") {
+            Click(locator1 = "ReportsAdminViewReportEntry#INFO_MESSAGE");
+            
+			Click.javaScriptClick(
+                key_value = "Select Data Set View",
+                locator1 = "Button#SECONDARY_BY_TITLE");
+            
+			SelectFrame(
+                key_title = "Select",
+                locator1 = "IFrame#MODAL_ANY");
+            
+			Click.clickAtNotVisible(
+                key_radioOption = "View Fields Test",
+                locator1 = "CommerceEntry#RADIO_BUTTON");
```

Comment:
This Commerce path has nothing to do with our test. Please use a generic button path from the Button.path file